src/components/elements/Button.jsx

```diff
@@ -1,7 +1,7 @@
 //  Available size options(String):
 //  - 'small'
 //  - 'medium'
-//  - 'big'
+//  - 'large'
```

Comment:
Take a look at JSDoc when you have time https://jsdoc.app/about-getting-started

When you use JSDoc comment format you will see hints in your editor. And your comments will become more readable.